sqrt/__init__.py

```diff
@@ -48,6 +49,8 @@ def get_data_stream(iterable):
 
 
 def main(save_to, num_batches, continue_=False):
+    if continue_:
+        continue_training(save_to)
```

Comment:
I think you need an `else:` following this line, or maybe we can change `continue_training` to return the `MainLoop` and do `return continue_training(save_to)` instead. The latter might actually be the best idea, because it allows people to train for a few epochs, and then interact with the main loop some more.